blitzstudios/sleeper-mini
# Converting socket connections from UDP -> TCP to improve stability.

### Description

This commit is part of a 4-part (4 repos) PR to change the socket event protocol from UDP to TCP for minis.

This allows us to maintain stateful connections, receive packets in order, and transmit much more data over the network during mini development.

See all PRs here:
[sleeper-mini](https://github.com/blitzstudios/sleeper-mini/pull/5)
[sleeper-mini-core](https://github.com/blitzstudios/sleeper-mini-core/pull/6)
[sleeperbot](https://github.com/blitzstudios/sleeperbot/pull/5433)
[repack](https://github.com/blitzstudios/repack/commit/f0b4a04cbb98394e2cd9b7ca1e61faf96a4bb3e0)

### How To Test

1. Load each branch locally.
2. Start the template project and Sleeper app in either order.
3. Make sure a connection is established.
4. Context sends and source code changes should all refresh as expected.
